dockers/base-cuda/Dockerfile

```diff
@@ -6,13 +6,15 @@
 # --build-arg TORCH_VERSION=1.2 --build-arg CUDA_VERSION=10.0
 # --build-arg TORCH_VERSION=1.1.0 --build-arg CUDA_VERSION=10.0 --build-arg CUDNN_VERSION=7.5
 
-ARG TORCH_VERSION=1.6.0
-ARG CUDA_VERSION=10.1
 ARG CUDNN_VERSION=7
+ARG CUDA_VERSION=10.1
 
-# TODO: make his imagge from pure Ubuntu + install all NVIDIA drivers
-# FROM nvidia/cuda:${CUDA_VERSION}-base
-FROM pytorch/pytorch:${TORCH_VERSION}-cuda${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel
+FROM nvidia/cuda:${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel
+
+ARG PYTHON_VERSION
+ARG PYTORCH_VERSION
+ARG PYTORCH_CHANNEL
```

Comment:
let's add defaults

Reply:
so defaults are pytorch 1.6 and python 3.7?

Reply:
I would say so, but you can put anything, it serves as an example for a user what he can add and what is the format...